examples/pointcloud/app.js

```diff
@@ -16,7 +16,7 @@ import {OBJLoader} from '@loaders.gl/obj';
 import {load, registerLoaders} from '@loaders.gl/core';
 
 import ControlPanel from './components/control-panel';
-import fileDrop from './components/file-drop';
+// import fileDrop from './components/file-drop';
```

Comment:
Do we need to some work to restore file drop?

Reply:
I will fix in another PR